cohere/client.py

```diff
@@ -192,6 +193,26 @@ def classify(
 
         return Classifications(classifications)
 
+
+    def extract(
+        self,
+        model: str,
+        extract: Extract,
```

Comment:
is this how we want to do this in the future? this is very different from how we do the other API calls

Reply:
yes, if we do it this way for extract, we probably wanna refactor for all

Reply:
i think i prefer 2 params instead of just 1 extract param

Reply:
`List[ExtractExample]` & `List[ExtractEntity]`